[2023-05-11T23:25:29.139+0530] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_storm_connection.connect_to_kafka_task manual__2023-05-11T10:41:13.807950+00:00 [queued]>
[2023-05-11T23:25:29.148+0530] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_storm_connection.connect_to_kafka_task manual__2023-05-11T10:41:13.807950+00:00 [queued]>
[2023-05-11T23:25:29.149+0530] {taskinstance.py:1331} INFO - Starting attempt 2 of 2
[2023-05-11T23:25:29.165+0530] {taskinstance.py:1350} INFO - Executing <Task(BashOperator): connect_to_kafka_task> on 2023-05-11 10:41:13.807950+00:00
[2023-05-11T23:25:29.169+0530] {standard_task_runner.py:57} INFO - Started process 11042 to run task
[2023-05-11T23:25:29.173+0530] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_storm_connection', 'connect_to_kafka_task', 'manual__2023-05-11T10:41:13.807950+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/kafka-storm con.py', '--cfg-path', '/tmp/tmpca0d4q1m']
[2023-05-11T23:25:29.175+0530] {standard_task_runner.py:85} INFO - Job 141: Subtask connect_to_kafka_task
[2023-05-11T23:25:29.223+0530] {task_command.py:410} INFO - Running <TaskInstance: kafka_storm_connection.connect_to_kafka_task manual__2023-05-11T10:41:13.807950+00:00 [running]> on host nizam-HP-15-Notebook-PC
[2023-05-11T23:25:29.297+0530] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_storm_connection' AIRFLOW_CTX_TASK_ID='connect_to_kafka_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-11T10:41:13.807950+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-11T10:41:13.807950+00:00'
[2023-05-11T23:25:29.299+0530] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-05-11T23:25:29.299+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'storm kafka -t test -b localhost:9092']
[2023-05-11T23:25:29.309+0530] {subprocess.py:86} INFO - Output:
[2023-05-11T23:25:29.426+0530] {subprocess.py:93} INFO - Unknown command: [storm kafka -t test -b localhost:9092]
[2023-05-11T23:25:29.426+0530] {subprocess.py:93} INFO - Commands:
[2023-05-11T23:25:29.427+0530] {subprocess.py:93} INFO - 	activate
[2023-05-11T23:25:29.427+0530] {subprocess.py:93} INFO - 	blobstore
[2023-05-11T23:25:29.427+0530] {subprocess.py:93} INFO - 	classpath
[2023-05-11T23:25:29.427+0530] {subprocess.py:93} INFO - 	deactivate
[2023-05-11T23:25:29.427+0530] {subprocess.py:93} INFO - 	dev-zookeeper
[2023-05-11T23:25:29.428+0530] {subprocess.py:93} INFO - 	drpc
[2023-05-11T23:25:29.428+0530] {subprocess.py:93} INFO - 	get-errors
[2023-05-11T23:25:29.428+0530] {subprocess.py:93} INFO - 	heartbeats
[2023-05-11T23:25:29.428+0530] {subprocess.py:93} INFO - 	help
[2023-05-11T23:25:29.429+0530] {subprocess.py:93} INFO - 	jar
[2023-05-11T23:25:29.429+0530] {subprocess.py:93} INFO - 	kill
[2023-05-11T23:25:29.429+0530] {subprocess.py:93} INFO - 	kill_workers
[2023-05-11T23:25:29.429+0530] {subprocess.py:93} INFO - 	list
[2023-05-11T23:25:29.429+0530] {subprocess.py:93} INFO - 	localconfvalue
[2023-05-11T23:25:29.429+0530] {subprocess.py:93} INFO - 	logviewer
[2023-05-11T23:25:29.430+0530] {subprocess.py:93} INFO - 	monitor
[2023-05-11T23:25:29.430+0530] {subprocess.py:93} INFO - 	nimbus
[2023-05-11T23:25:29.430+0530] {subprocess.py:93} INFO - 	node-health-check
[2023-05-11T23:25:29.430+0530] {subprocess.py:93} INFO - 	pacemaker
[2023-05-11T23:25:29.430+0530] {subprocess.py:93} INFO - 	rebalance
[2023-05-11T23:25:29.430+0530] {subprocess.py:93} INFO - 	remoteconfvalue
[2023-05-11T23:25:29.430+0530] {subprocess.py:93} INFO - 	repl
[2023-05-11T23:25:29.431+0530] {subprocess.py:93} INFO - 	set_log_level
[2023-05-11T23:25:29.431+0530] {subprocess.py:93} INFO - 	shell
[2023-05-11T23:25:29.431+0530] {subprocess.py:93} INFO - 	sql
[2023-05-11T23:25:29.431+0530] {subprocess.py:93} INFO - 	supervisor
[2023-05-11T23:25:29.431+0530] {subprocess.py:93} INFO - 	ui
[2023-05-11T23:25:29.431+0530] {subprocess.py:93} INFO - 	upload-credentials
[2023-05-11T23:25:29.431+0530] {subprocess.py:93} INFO - 	version
[2023-05-11T23:25:29.431+0530] {subprocess.py:93} INFO - 
[2023-05-11T23:25:29.432+0530] {subprocess.py:93} INFO - Help:
[2023-05-11T23:25:29.432+0530] {subprocess.py:93} INFO - 	help
[2023-05-11T23:25:29.432+0530] {subprocess.py:93} INFO - 	help <command>
[2023-05-11T23:25:29.432+0530] {subprocess.py:93} INFO - 
[2023-05-11T23:25:29.432+0530] {subprocess.py:93} INFO - Documentation for the storm client can be found at http://storm.apache.org/documentation/Command-line-client.html
[2023-05-11T23:25:29.432+0530] {subprocess.py:93} INFO - 
[2023-05-11T23:25:29.432+0530] {subprocess.py:93} INFO - Configs can be overridden using one or more -c flags, e.g. "storm list -c nimbus.host=nimbus.mycompany.com"
[2023-05-11T23:25:29.432+0530] {subprocess.py:93} INFO - 
[2023-05-11T23:25:29.436+0530] {subprocess.py:97} INFO - Command exited with return code 254
[2023-05-11T23:25:29.446+0530] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 254.
[2023-05-11T23:25:29.450+0530] {taskinstance.py:1368} INFO - Marking task as FAILED. dag_id=kafka_storm_connection, task_id=connect_to_kafka_task, execution_date=20230511T104113, start_date=20230511T175529, end_date=20230511T175529
[2023-05-11T23:25:29.465+0530] {standard_task_runner.py:104} ERROR - Failed to execute job 141 for task connect_to_kafka_task (Bash command failed. The command returned a non-zero exit code 254.; 11042)
[2023-05-11T23:25:29.505+0530] {local_task_job_runner.py:232} INFO - Task exited with return code 1
[2023-05-11T23:25:29.529+0530] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
